index.html

```diff
@@ -1,35 +1,99 @@
+                            <a href="http://2018.igem.org/Team:Cornell/Sponsors">SPONSORS</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Attributions">ATTRIBUTIONS</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">OUTREACH</button>
+                        <div class="dropdown-content outreach-dropdown-content">
+                            <a href="#">COLLABORATIONS</a>
+                            <a href="#">PUBLIC ENGAGEMENT</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">HUMAN-CENTERED DESIGN</button>
+                        <div class="dropdown-content human-dropdown-content">
+                            <a href="#">PRACTICES</a>
+                            <a href="#">POLICIES</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Entrepreneurship">ENTREPRENEURSHIP</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">MODELING</button>
+                        <div class="dropdown-content modeling-dropdown-content">
+                            <a href="http://2018.igem.org/Team:Cornell/Model">MODEL</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Software">SOFTWARE</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Design">DESIGN</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">TOOLKIT</button>
+                        <div class="dropdown-content toolkit-dropdown-content">
+                            <ul>
+                                <div class="nav-first-col">
+                                    <li><a href="#"><b>WET LAB</b></a>
+                                        <ul>
+                                            <li><a href="http://2018.igem.org/Team:Cornell/Foundations">FOUNDATIONS</a></li>
+                                            <li><a href="http://2018.igem.org/Team:Cornell/Demonstrate">DEMONSTRATE</a></li>
+                                            <li><a href="http://2018.igem.org/Team:Cornell/InterLab">INTERLAB</a></li>
+                                            <li><a href="hhttp://2018.igem.org/Team:Cornell/Parts">PARTS</a></li>
+                                        </ul>
+                                    </li>
+                                </div>
 
-	<!------------------------ HOME BANNER START ------------------------>
-	<header class="home-banner">
-		<svg class="red-wave-animation" viewbox="0 0 1100 1100" width="110%">
+                                <div class="nav-second-col">
+                                    <li><a href="#"><b>DOCUMENTATION</b></a>
```

Comment:
Same thing with WET LAB. Can you make it so that hovering over DOCUMENTATION doesn't change color? So there should be no white button when you hover over DOCUMENTATION.